Package.swift

```diff
@@ -24,14 +24,20 @@ let package = Package(
         .package(url: "https://github.com/RougeWare/Swift-Rectangle-Tools.git", from: "2.9.0"),
         .package(url: "https://github.com/RougeWare/Swift-Cross-Kit-Types.git", from: "1.0.0"),
         .package(url: "https://github.com/RougeWare/Swift-Optional-Tools.git",  from: "1.0.0"),
+        .package(url: "https://github.com/RougeWare/Swift-Simple-Logging.git",  from: "0.4.4"),
```

Comment:
Major version bump - anything which depends on this cannot also depend on SimpleLogging 0.3, 0.5, 1.0, etc., nor any packages which depend on one of those